src/storage-controller/src/lib.rs

```diff
@@ -2500,14 +2550,18 @@ where
             }
             // If collection is closed or the frontier is the minimum, we cannot
             // or don't need to truncate (respectively).
-            _ => return,
+            _ => return None,
```

Comment:
return empty btree?

Reply:
This was just reinstating some old code that was here before––will fix this up to just return the empty bree!

Reply:
thanks!